store/server/features/timesheet/dashboard/queries.ts

```
@@ -0,0 +1,192 @@
+export const useGetAdminAttendanceStats = (params: {
+    startDate: string;
+    endDate: string;
+    departmentId?: string;
+}) => {
+    return useQuery(['adminAttendanceStats', params], () =>
+        getAdminAttendanceStats(params),
+    );
+};
+
+export const getAdminAttendanceUsers = async (params: {
+    sortBy: string;
```

ESLint

[error] Delete `··`

(prettier/prettier)